pkgs/misc/vim-plugins/vim-plugin-names

```diff
@@ -63,6 +63,7 @@ buoto/gotests-vim
 camspiers/lens.vim
 camspiers/snap
 carlitux/deoplete-ternjs
+catppuccin/nvim as catppuccin-nvim
```

Comment:
Do we support this syntax? Seeing it the first time right now.

Reply:
Dracula theme uses it too, since the github repo is just called nvim/vim.